RELEASE_NOTES.rst

```diff
@@ -4,6 +4,7 @@ Next release
 All changes
 -----------
 
+- `#243 <https://github.com/iiasa/ixmp/pull/243>`_: Add bulk timeseries data export
```

Comment:
Link to the docs for the new method, and check that the link works in the docs build. End with a period.
```
Add :meth:`.export_timeseries_data` to write data for multiple TimeSeries objects to CSV.
```